src/Components/Publishing/index.tsx

```diff
@@ -39,7 +39,7 @@ import {
   StandardArticle,
 } from "./Fixtures/Articles"
 
-export default {
+export const components = {
```

Comment:
Do you really want this to be named `components`? You’d have to use it like so:

```js
import { components } from "@artsy/reaction/dist/Components/Publishing"
const { Article } = components
```

You can use this ES6 syntax to just export the individual items:

```js
export { Article } from "./Article"
```

And then the import looks like:

```js
import { Article } from "@artsy/reaction/dist/Components/Publishing"
```

Reply:
Ah yeah the latter is definitely what we want. Good call!